static/demo/map-a-demo.html

```diff
@@ -0,0 +1,185 @@
23 -79.2315517 46.8527907 -79.2319334 46.8543815 -79.2323538 46.855713 -79.2329257 46.8573838 -79.233406 46.8584442 -79.233825 46.8592528 -79.2344899 46.8601991 -79.2350085 46.860756 -79.2360914 46.8617553 -79.2380905 46.8636017 -79.2389294 46.8644447 -79.2401198 46.8657302 -79.2417677 46.8675117 -79.2432017 46.8690643 -79.2442174 46.8701058 -79.2457048 46.8715324 -79.2478642 46.8735963 -79.2501457 46.8757936 -79.2510979 46.8766898 -79.2513817 46.8769799 -79.2526085 46.878414 -79.2548287 46.8810653 -79.2565923 46.8829728 -79.2570337 46.8834495 -79.2575459 46.8839341 -79.2580562 46.884365 -79.2600251 46.8859253 -79.2616814 46.8871919 -79.2623603 46.8877069 -79.2633813 46.8884314 -79.2649312 46.889473 -79.2657474 46.890007 -79.2668603 46.8906097 -79.2686924 46.8916093 -79.2716372 46.8931809 -79.2754289 46.8952179 -79.2781297 46.8966561 -79.2810287 46.8982353 -79.281883 46.898735 -79.2823559 46.8991317 -79.2827754 46.8995323 -79.2832558 46.900009 -79.2837825 46.9005012 -79.2842108 46.9010011 -79.2845378 46.9014663 -79.2848355 46.9019928 -79.286163 46.9044341 -79.2870486 46.9059754 -79.2925575 46.9144898 -79.292595 46.9145202 -79.2926639 46.9146386 -79.292809 46.9147912 -79.2929079 46.9149285 -79.2930299 46.9150849 -79.2931448 46.9152375 -79.2932434 46.9153939 -79.2933724 46.915531 -79.2935108 46.9157296 -79.2936625 46.9159011 -79.2938381 46.9161033 -79.2939838 46.9162865 -79.2941971 46.916523 -79.2943495 46.9167519 -79.294548 46.9169693 -79.2947156 46.9171524 -79.2949444 46.9173698 -79.2952343 46.9176444 -79.2954555 46.9178733 -79.2956465 46.9181022 -79.2958838 46.9183465 -79.2960969 46.918602 -79.2963558 46.9188422 -79.2965463 46.9190864 -79.2967076 46.9192315 -79.2969901 46.9194718 -79.2973256 46.9197274 -79.2976604 46.9199943 -79.297859 46.9202117 -79.2981795 46.9204444 -79.2984996 46.9207191 -79.2987816 46.9209288 -79.2991338 46.9211846 -79.2995155 46.9214975 -79.2999274 46.9217645 -79.300362 46.9220429 -79.30075 46.9222792 -79.3010715 46.9225274 -79.30146 46.9227485 -79.3018564 46.9229621 -79.302192 46.9231948 -79.302384 46.9233208 -79.3025902 46.923504 -79.3029246 46.9236258 -79.3031993 46.923828 -79.3035735 46.9240608 -79.3040079 46.9242896 -79.3044358 46.9245224 -79.3049158 46.9248008 -79.3053516 46.9250031 -79.305718 46.9251977 -79.3061144 46.9254112 -79.3064878 46.9256782 -79.3069149 46.9259452 -79.3073501 46.9261856 -79.3077385 46.92638 -79.3080369 46.9265633 -79.3082586 46.9267045 -79.308685 46.9269142 -79.3090436 46.9270935 -79.3094179 46.9273033 -79.3098979 46.9275817 -79.310349 46.9279023 -79.3106691 46.9281502 -79.3110126 46.9283982 -79.3113091 46.9286917 -79.3116072 46.9291 -79.3118897 46.9293862 -79.3122178 46.9297218 -79.3124843 46.9300422 -79.3126911 46.9302789 -79.3127447 46.9303247 -79.3131864 46.9307441 -79.3133767 46.9309615 -79.3136449 46.9312821 -79.3139191 46.931591 -79.3141477 46.9318275 -79.3144985 46.9321746 -79.3148263 46.9325065 -79.3151696 46.9327964 -79.3155449 46.9330903 -79.3159791 46.9334755 -79.3163454 46.933921
```

Comment:
This link is 404.

Reply:
This was a LHIN boundary I think.  So the link should go here: https://www.ottawapublichealth.ca/

Reply:
But maybe we should think about making the demo go from Canada to the provinces layer i.e. I think the link from Ontario to Ottawa is good, but would make more sense from Ontario to whatever breakdown of Ontario we were mapping e.g. the LHIN boundaries or Census Divisions, perhaps.  The problem with *that* is the volume of data we have to include in the demo player.  Needs a bit more design thought.

Reply:
Even a polygon to a set of point markers would be reasonable

Reply:
Updated the demo's for map-a and map-feature, please comment if any changes need to be made! Thanks!